test/Joonasw.AspNetCore.SecurityHeaders.Samples/appsettings.json

```diff
@@ -1,10 +1,15 @@
 ﻿{
-  "Logging": {
-    "IncludeScopes": false,
-    "LogLevel": {
-      "Default": "Debug",
-      "System": "Information",
-      "Microsoft": "Information"
+    "Logging": {
+        "IncludeScopes": false,
+        "LogLevel": {
+            "Default": "Debug",
+            "System": "Information",
+            "Microsoft": "Information"
+        }
+    },
+    "Hsts": {
+        "Duration": "30",
```

Comment:
Could you change this duration to `"30:00:00:00"`? Since `Duration` is a `TimeSpan`, it is good to show that you can specify the days, hours, minutes and seconds.